.github/workflows/soak.yml

```diff
@@ -455,20 +455,6 @@ jobs:
           name: pr-number
           path: /tmp/pr.txt
 
-      - name: Read analysis file
-        id: read-analysis
-        uses: juliangruber/read-file-action@v1
-        with:
-          path: /tmp/${{ github.run_id }}-${{ github.run_attempt }}-analysis
-
-      - name: Post Results To PR
-        uses: peter-evans/create-or-update-comment@v1
-        continue-on-error: true
-        with:
-          issue-number: ${{ needs.compute-soak-meta.outputs.pr-number }}
-          edit-mode: replace
-          body: ${{ steps.read-analysis.outputs.content }}
-
```

Comment:
Sorry - where are the ones we're keeping that these duplicate?

Reply:
Ah in `soak_comment.yml`.

Reply:
Oh right - separate file, thanks!